src/documents/tutorial-userInterfaceOptions/UsingImagesInContrastModes.md

```diff
@@ -0,0 +1,155 @@
+
+The simplest images to convert are ones that already have a transparent background. Creating contrast versions involves 'locking' the transparent pixels so they stay transparent and then painting over everything not-transparent with the desired colour.
+
+### Step 1 - Lock the transparent pixels
+
+In Photoshop, you can lock the transparent pixels in one of at least two ways:
+
+1. Type the '/' key, or
+2. In the "Layers" panel, click the 'lock pixels button (see image below).
+
+![Image of the Photoshop Layers panel with the mouse cursor hovering over the Lock transparent pixels button]
+
+### Step 2 - Fill the rest of the image with the desired colour
+
+In Photoshop, there are several ways to fill the image with a single colour:
+
+1. Type Alt + Delete on Mac, or Alt + Backspace on Windows to fill the image with the current foreground colour, or
+2. Type Command + Delete on Mac, or Command + Backspace on Windows to fill the image with the current background colour, or
+3. Choose a paintbrush and a foreground colour, and drag the cursor over the image.
+
+### Step 3 - Save the image for the web
+
+* On Mac: Type Shift + Alt + Command + S
+* On Windows: Type Shift + Alt + Ctrl + S
+
+### Inverting Colours for Quick Black-White and White-Black Images
+
+If you've created a black (or white) image, you can invert the colour to convert it to white (or black) by typing Command + I on Mac, or Ctrl + I on Windows.
+
+## Using CSS to Switch Web Graphics for Contrast Modes
+
+While web images and graphics are typically implemented using `<img>` tags, using background images instead makes it very simple to substitute a different version of the image when a high contrast theme is used.
+
+### When to Use Background Images and `<img>` Tags
+
+Images on web pages are often implemented by using `<img>` tags. However, the `background` CSS property for images may be advantageous.
+
+To help decide whether to use an `<img>` element or a background image, a good rule of thumb would be to use:
+* `<img>` tags if the graphic is important as content (i.e. a graph, photo, or a diagram).
+* CSS `background` property for graphics that are cosmetic in nature, such as icons and logos.
+
+The advantage of using `background` is that assistive technologies (like screen readers) will ignore them since it is considered as cosmetic. This allows you to declutter and simplify the user experience by not having extraneous content.
+
+<div class="infusion-docs-note"><strong>Note:</strong> UI Options changes the presentation of content by using CSS. Therefore images implemented with &lt;img&gt; HTML elements are not easily modified using CSS. It is possible to achieve a similar contrast effect by using <a href="https://developer.mozilla.org/en/docs/Web/CSS/filter">CSS filters</a>, but the results may not match the contrast mode.</div>
```

Comment:
the `<img>` tag should be in a `<code>` element so that it is appropriately highlighted.